api/tasks/runner_test.go

```diff
@@ -0,0 +1,67 @@
+package tasks
+
+import (
+	"testing"
+	"math/rand"
+	"time"
+	"os"
+)
+
+
+func TestCheckTmpDir(t *testing.T) {
+
+	//It should be able to create a random dir in /tmp
+	dirName := os.TempDir()+ "/" + randString(rand.Intn(10 - 4) + 4)
```

Comment:
Why you're don't use the simple https://golang.org/pkg/io/ioutil/#TempDir? As I see, it creates the random name (after prefix) too.

Reply:
Because I dont want to create the directory at this point. I want to test that my function is able to create a directory when it does not exist